### DISCUSSION TITLE
post/2023-aws-reinvent-1/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#57] [Announcements]
# post/2023-aws-reinvent-1/

AWS re:Invent 2023에 참여하며 관심 주제를 중심으로 참석한 세션을 소개하고, 느낀 점을 공유하고자 합니다.

https://tech.kakaopay.com/post/2023-aws-reinvent-1/

**Comment by yrnana** [comment]
좋은 경험과 지식 공유해주셔서 감사합니다~~!!

**Comment by Nokako** [comment]
갓록 고생하셨습니다~
